skype/skype-ps/skype/Grant-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/grant-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Grant-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Assigns a Teams audio-conferencing policy at the per-user scope. Audio conferencing policies are used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+### GrantToTenant (Default)
+```
+Grant-CsTeamsAudioConferencingPolicy [-Global] [-PassThru] [[-PolicyName] <String>] [-WhatIf] [-Confirm]
```

Comment:
What about MsftInternalProcessingMode? If Docs readers see it in the code, they can ask about it.

Reply:
This is not exposed.

Reply:
Yes, it's exposed in 4.2.0. If you will remove it in the next version it's ok.